app/scripts/move-popup/dimMovePopup.directive.js

```diff
@@ -21,7 +21,7 @@
         '  <dim-move-amount ng-if="vm.item.amount > 1 && !vm.item.notransfer" amount="vm.moveAmount" maximum="vm.maximum"></dim-move-amount>',
         '  <div class="interaction">',
         '    <div class="locations" ng-repeat="store in vm.stores | sortStores:vm.settings.characterOrder track by store.id">',
-        '      <div class="move-button move-vault" alt="{{::store.name}}" title="{{::store.name}}" ',
+        '      <div class="move-button move-vault" translate-attr="{ title: \'Bucket.Vault\', alt: \'Bucket.Vault\' }" ',
```

Comment:
Looks like it's because it used bind-once? Not sure. `store.name` should be translated.